onmt/dynamic/get_vocab.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""Get vocabulary coutings from transformed corpora samples."""
+from onmt.utils.logging import init_logger
+from onmt.utils.misc import set_random_seed
+from onmt.dynamic.parse import DynamicArgumentParser
+from onmt.dynamic.opts import dynamic_prepare_opts
+from onmt.dynamic.corpus import save_transformed_sample
+from onmt.dynamic.transforms import make_transforms, get_transforms_cls
+
```

Comment:
Also, move it to onmt.bin

Reply:
I just realize that this does not need to be moved into bin as it's not a necessary step in our settings, same for the .rst file related.

Reply:
I think it makes more sense to move it to bin and make the corresponding entrypoint in setup.py.
So that a user with very basic knowledge can execute a pipeline simply by calling onmt_build_vocab & onmt_train, like they were before by calling onmt_preprocess & onmt_train.